modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -57,3 +57,33 @@ The Operator is installed automatically into the `openshift-operators` namespace
 ====
 +
 . Verify that the *Status* is set to *Succeeded Up to date*  to confirm successful installation of {pipelines-title} Operator.
++
+[WARNING]
+====
+The success status may show as *Succeeded Up to date* even if installation of other components is in-progress. Therefore, it is important to verify the installation manually in the terminal.
+====
```

Comment:
I think you can remove the text on line 66 and just have the instruction for step 7 state: "Verify that all components of the Red Hat OpenShift Pipelines Operator were installed successfully. Login to the cluster on the terminal, and run the following commands:"